Modellparameter/MotorParams_Franke97.m

```diff
@@ -14,11 +14,15 @@
     % Motorreibungskonstanten
     stM.d0_MG = 0.052; % konstante Reibung in Nm
     stM.dw_MG = 0.0000574; % geschwindigkeitsabhängige Reibung in Nm/(rad/s)
-
+    
+    % Steuerspannung
     stM.Umin = -10;
     stM.Umax = 10;
-
-    % Dummy-Werte
+    
+    % Maximale Stellkraft
+    % -> Max. Impustrom ist 20A, Drehmomentkonstante 0,153, Getriebe i=60/16,
+    %    Antriebsradius r=0,0255
+    % Fmax = 20*0,153*60/16/0,0255 = 420,75 < 421
     stM.Fmin = -421;

```

Comment:
Gut es passt, würde aber Fmax aus dem Modell rausnehmen, da es ja keine echte Kraftbegrenzung gibt

Reply:
Ja, guter Punkt. Ich würde unser Modell allerdings generell umdenken. Daher lasse ich den Motorblock jetzt erstmal in Ruhe, bis klar ist, wie das passiert.